src/System.Drawing.Common/src/System/Drawing/Font.cs

```diff
@@ -77,18 +77,11 @@ public sealed partial class Font : MarshalByRefObject, ICloneable, IDisposable,
         /// Gets the face name of this <see cref='Font'/> .
         /// </summary>
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
-#if !NETCORE
```

Comment:
@safern @ViktorHofer do you think this broke mono consumption of System.Drawing sources?  Same below.  /cc @JuditRose

Reply:
I did check if these files where included in the Mono sources and it seems like they're not:

https://github.com/mono/mono/blob/master/mcs/class/Facades/System.Drawing.Common/System.Drawing.Common.dll.sources

Reply:
Got it, so these were simply dead-code ifdefs.  That's good to know.  @JuditRose from that perspective you can clean this up with your change.

Reply:
Some of the files are used in https://github.com/mono/mono/blob/master/mcs/class/System.Drawing/System.Drawing.dll.sources but it should be fine since we stopped taking corefx updates for traditional Mono anyway due to the focus on .NET 5.

Reply:
@ericstj @safern here's the answer to the question that I raised in yesterday's standup.